libs/perun/services/src/lib/auth.service.ts

```diff
@@ -291,4 +292,16 @@ export class AuthService {
   public getIdpFilter(): string {
     return this.filterShortname;
   }
+
+  private refreshAndStoreToken(): Promise<boolean> {
+    return this.oauthService.refreshToken().then(
+      (response) => {
+        localStorage.setItem('refresh_token', response['refresh_token']);
+        return true;
+      },
+      () => {
+        return false;
+      }
```

Comment:
I think it looks good. There is just one eslint error.
```suggestion
      () => false
```